webpack-modules/webpack.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    entry: {
+        background_scripts: "./background_scripts/background.js",
+        popup: "./popup/left-pad.js"
+    },
+    output: {
+        path: "dist",
```

Comment:
This can be whatever, so sure, we can make it "addon". By the way, if it's not marked in the README yet, I should probably add that.